src/calculateRentalCost.js

```diff
@@ -5,6 +5,27 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRent = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  const totalRentPrice = days * dailyRent;
+
+  if (days === 0) {
```

Comment:
Checking for `days === 0` is not necessary if you assume that the function will always be called with a positive number of days. If you want to handle incorrect inputs, consider throwing an error or handling the case where `days` is less than 0.